mandatory/3-function-output.js

```diff
@@ -1,14 +1,17 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// return a radom number between 1 and 10
```

Comment:
its between 0 and 10 actually

Reply:
yes, thanks again... surprising how it's hard to detect some of these things